rig-core/Cargo.toml

```diff
@@ -50,6 +52,7 @@ pdf = ["dep:lopdf"]
 epub = ["dep:epub", "dep:quick-xml"]
 rayon = ["dep:rayon"]
 worker = ["dep:worker"]
+blob_data = ["dep:base64", "dep:mime_guess"]
```

Comment:
`blob_data` is a weird feature name? maybe just `blob`? i'm not sure, any ideas @cvauclair ?